internal/bminventory/inventory.go

```diff
@@ -3734,7 +3734,9 @@ func (b *bareMetalInventory) GetCredentialsInternal(ctx context.Context, params
 		log.WithError(err).Errorf("failed to find cluster %s", params.ClusterID)
 		return nil, err
 	}
-	if !b.clusterApi.IsOperatorAvailable(&cluster, operators.OperatorConsole.Name) {
+	isConsoleMonitored := b.clusterApi.IsOperatorMonitored(&cluster, operators.OperatorConsole.Name)
+	isConsoleAvailable := b.clusterApi.IsOperatorAvailable(&cluster, operators.OperatorConsole.Name)
+	if isConsoleMonitored && !isConsoleAvailable {
```

Comment:
Just in matter of running less code though it can be less readable. 

if isConsoleMonitored && !b.clusterApi.IsOperatorAvailable(&cluster, operators.OperatorConsole.Name)

Reply:
I think it is actually a little bit wrong. In case we don't have console why we need to run this logic?
This code returns 
`&models.Credentials{
		Username:   DefaultUser,
		Password:   string(password),
		ConsoleURL: common.GetConsoleUrl(cluster.Name, cluster.BaseDNSDomain),
	}`
and console url is part of it but console is not enabled so what those creds should be used for?

Reply:
The password is used to create the "kubeadmincreds" secret. The console URL is returned by the API and I assume it is used by the UI. I don't see it used in any other place in this project.

I am updating the code so that the `ConsoleURL` field is only populated when the console is enabled. Please check again.